src/crystal/system/win32/time.cr

```diff
@@ -1,9 +1,33 @@
+require "c/time"
+
 module Crystal::System::Time
+  WINDOWS_EPOCH_IN_SECONDS = 50491123200_i64
+
   def self.compute_utc_offset(seconds : Int64) : Int32
-    raise NotImplementedError.new("Crystal::System::Time.compute_utc_offset")
+    ret = LibC.get_time_zone_information(out tz)
+    raise Errno.new("GetTimeZoneInformation") if ret == -1
+
+    tz.bias.to_i32 * -60
   end
 
   def self.compute_utc_seconds_and_nanoseconds : {Int64, Int32}
-    raise NotImplementedError.new("Crystal::System::Time.compute_utc_seconds_and_nanoseconds")
+    LibC.get_system_time_as_file_time(out filetime)
+    since_epoch = (filetime.high_date_time.to_u64 << 32) | filetime.low_date_time.to_u64
+
+    {(since_epoch / 10_000_000).to_i64 + WINDOWS_EPOCH_IN_SECONDS, since_epoch.remainder(10_000_000).to_i32 * 100}
+  end
+
+  @@performance_frequency = uninitialized Int64
```

Comment:
```cr
@@performance_frequency = begin
  ret = LibC._QueryPerformanceFrequency(out frequency)
  raise Errno.new("QueryPerformanceFrequency") if ret == 0
  frequency
end
```

should work.

Reply:
Or just a plain:
```crystal
ret = LibC.QueryPerformanceFrequency(out frequency)
raise Errno.new("QueryPerformanceFrequency") if ret == 0
@@performance_frequency = frequency
```

Reply:
Or should it be lazy evaluated like `mach_timebase_info` for unix/macos?

Reply:
I would prefer @RX14's version so you dont have a `ret` variable floating around